pages/docs/getting-started/quick-start/python.mdx

```diff
@@ -8,7 +8,9 @@ export const description =
 This guide will teach you how to add Inngest to a FastAPI app and run an Inngest function.
 
 <Callout variant="info">
-  💡 Inngest currently supports DigitalOcean Functions, Django, FastAPI, Flask, and Tornado. If you'd like support for your favorite framework please open an issue on [GitHub](https://github.com/inngest/inngest-py)!
+  💡 If you prefer to explore code instead, here are example apps in the frameworks currently supported by Inngest: [DigitalOcean Functions](https://github.com/inngest/inngest-py/tree/main/examples/digital_ocean), [Django](https://github.com/inngest/inngest-py/tree/main/examples/django), [FastAPI](https://github.com/inngest/inngest-py/tree/main/examples/fast_api), [Flask](https://github.com/inngest/inngest-py/tree/main/examples/flask), and [Tornado](https://github.com/inngest/inngest-py/tree/main/examples/tornado). 
```

Comment:
```suggestion
  💡 If you prefer to explore code instead, here are example apps in the frameworks currently supported by Inngest:  [FastAPI](https://github.com/inngest/inngest-py/tree/main/examples/fast_api), [Django](https://github.com/inngest/inngest-py/tree/main/examples/django),  [Flask](https://github.com/inngest/inngest-py/tree/main/examples/flask), [DigitalOcean Functions](https://github.com/inngest/inngest-py/tree/main/examples/digital_ocean), and [Tornado](https://github.com/inngest/inngest-py/tree/main/examples/tornado). 
```

suggestion: I know it's not alphabetical, but I think sorting them in terms of what is in this quick start then rough popularity is good b/c Digital Ocean functions aren't as widely used.

Reply:
tbh I also thought about it so thanks for validating this